Review.java

```diff
@@ -80,9 +84,83 @@ public static String textToString( String fileName )
       System.out.println("Unable to locate " + fileName);
     }
     //make sure to remove any additional space that may have been added at the end of the string.
+    //totalSentiment("SimpleReview.txt");
     return temp.trim();
+  }
+     /**
+   * Returns the word after removing any beginning or ending punctuation
+   */
+  public static String removePunctuation( String word )
+  {
+    while(word.length() > 0 && !Character.isAlphabetic(word.charAt(0)))
+    {
+      word = word.substring(1);
+    }
+    while(word.length() > 0 && !Character.isAlphabetic(word.charAt(word.length()-1)))
+    {
+      word = word.substring(0, word.length()-1);
+    }
+    
+    return word;
+  }
+
+  
+  public static double totalSentiment(String fileName)
+  {
+   //open file
+   //pick each word -> when I find a space, all the stuff before was a word. But I've got to remove punctuation.
+   //look up sentiment value
+   //add to total
+   //return value
+      String file = textToString(fileName);
+      String word = "";
+      String space = " ";
+      double totalVal = 0;
+      for(int i = 0; i < file.length(); i++)
+      {
+         removePunctuation(fileName);
+         String str = file.substring(i, i+1);
+         if(str.equals(space))
+         {
+            totalVal += sentimentVal(word);
+            word = "";
+         }
+         else
+         {
+            word += str;
+         }
+      }
+      return totalVal;
+      
+      
   }
   
+  public static int starRating(String fileName);
+  {
+      String rate = totalSentiment(fileName);
+         if(rate > 10)
+         {
+            return 4;
+         }
+         else if(rate < 10 && rate > 5)
+         {
+            return 3;
+         }
+         else if(rate < 5 && rate > 0)
+         {
+            return 2;
+         }
+         else if(rate <= 0)
+         {
+            return 1;
+         }
+         else
+         {
+            return 0;

```

Comment:
I don't believe there would ever be an instance where this else statement would be used. If you want a 5 point scale, maybe add another condition (a lower limit) to the last else if statement (on line 153)